### ISSUE TITLE
Add `mpc`

### ISSUE THREAD

**Issue by LoganDark** [kiss-community/community#1158]
`mpc` is required to build certain configurations of the Linux kernel.

Here's a working package, I would PR it but I don't currently have my GPG key or git setup on Linux.

[mpc-1.12.1-1.tar.gz](https://github.com/kiss-community/community/files/9107992/mpc-1.12.1-1.tar.gz)

**Comment by dilyn-corner** [comment]
Best bet for inclusion is 

1) a commitment to maintain the package 

2) submitting this as a patch
Need not be submitted as a git formatted patch, as long as we know you're maintaining it. 


Patches are the preferred way to submit updates in lieu of a PR

Jul 13, 2022, 10:50 PM by ***@***.***:

**Comment by LoganDark** [comment]
> 1) a commitment to maintain the package 

At this point I'm committed to KISS Linux, even though I can't get it to do [the ephemeral thing I like](https://www.reddit.com/r/linuxmasterrace/comments/vtwoip/any_distros_that_are_clean_like_alpine_linux/), it seems like what I want in general. (I'm using GKISS specifically, since musl libc is exactly what I'm trying to escape.)

> 2) submitting this as a patch
> Need not be submitted as a git formatted patch, as long as we know you're maintaining it.

Should I just format the existing .tar.gz as a patch? I mean, applying the patch would be completely identical to untarring the archive, but I guess if you have format requirements...

**Comment by ioraff** [comment]
how goes it?

**Comment by LoganDark** [comment]
I've received zero responses to any of my calls for help with my trackpad on Reddit and Discord, and on the former one of my posts was even caught in the spam filter and the moderation team hasn't responsed in 3 days. Going to try an allyesconfig build soon.

TL;DR my KISS system still does not work, I don't have much of a clue on how to fix it, and literally nobody is trying to help me. :(

**Comment by dilyn-corner** [comment]
We have a Discord? 
There's nothing pending in the mod queue on the subreddit.

I'd suggest IRC

**Comment by LoganDark** [comment]
> We have a Discord? There's nothing pending in the mod queue on the subreddit.
> 
> I'd suggest IRC

Not in KISS-specific Discords or subreddits. This isn't a KISS-specific issue - I'm running the mainline Linux kernel (well ok, Zen kernel, but I built mainline with the same config and it made zero difference). Would going to a KISS-specific venue help at all?

**Comment by dilyn-corner** [comment]
It might, just because most of the people running around have been through something like this and can usually quickly pinpoint what's missing.

**Comment by LoganDark** [comment]
I went ahead and crossposted to the KISSLinux subreddit just in case. As for IRC, well, [wouldn't be much use there right now](https://morethanone.info) due to personal complications that only expire September 12th.
